gqlgen.yml

```diff
@@ -25,6 +25,9 @@ models:
     model: github.com/evergreen-ci/evergreen/graphql.StringMap
   Patch:
     model: github.com/evergreen-ci/evergreen/rest/model.APIPatch
+    fields:
```

Comment:
Since the tab is going to conditionally display on the task page rather than the patch page, I think you actually want this as a field on Task